app/config.py

```diff
@@ -183,6 +183,8 @@ class Config(object):
     MOU_SIGNED_ON_BEHALF_SIGNER_RECEIPT_TEMPLATE_ID = "c20206d5-bf03-4002-9a90-37d5032d9e84"
     MOU_SIGNED_ON_BEHALF_ON_BEHALF_RECEIPT_TEMPLATE_ID = "522b6657-5ca5-4368-a294-6b527703bd0b"
     ORGANISATION_HAS_NEW_GO_LIVE_REQUEST_TEMPLATE_ID = "5c7cfc0f-c3f4-4bd6-9a84-5a144aad5425"
+    ORGANISATION_CONTINUE_GO_LIVE_REQUEST_TEMPLATE_ID = "62f12a62-742b-4458-9336-741521b131c7"
+    ORGANISATION_REJECTED_GO_LIVE_REQUEST_TEMPLATE_ID = "507d0796-9e23-4ad7-b83b-5efbd9496866"
```

Comment:
[naming suggestion] I would rename those constants a bit, so:
- domain comes first, here domain is the go-live-request
- we know who rejected the request
- we know who gets the email about next steps.

So the structure would be [domain]-[action]-[how org user relates to this action]

`GO_LIVE_REQUEST_REJECTED_BY_ORG_USER_TEMPLATE_ID`

`GO_LIVE_REQUEST_NEXT_STEPS_FOR_ORG_USER_TEMPLATE_ID`

Reply:
I will make that change. Will need to change the one above also, which is about go live requests too.